zebrad/src/components/mempool/error.rs

```diff
@@ -43,4 +43,7 @@ pub enum MempoolError {
     /// The queue's capacity is [`super::downloads::MAX_INBOUND_CONCURRENCY`].
     #[error("transaction dropped because the queue is full")]
     FullQueue,
+
+    #[error("mempool is disabled since synchronization did not reach the tip")]
```

Comment:
Message clarification:

Zebra might have reached the tip in the past, but it's not at it right now.
(For example, if it synced to the tip, but then disconnected from the network.)

```suggestion
    #[error("mempool is disabled since synchronization is behind the chain tip")]
```

Reply:
Done in 5244db346 along with other changes